roles/ovirt-recover/defaults/main.yml

```diff
@@ -0,0 +1,14 @@
+# Should perform a recover of the setup using the ini file.
+recovered: "False"
```

Comment:
Can we please use `dr_` prefix for all variables?

Reply:
done